plugin.video.drnu/addon.py

```diff
@@ -1,6 +1,5 @@
-#
-#      Copyright (C) 2014 Tommy Winther
-#      http://tommy.winther.nu
```

Comment:
The old coprytight infor should be kept. You can add yourself like you already did.

Reply:
Re-added - But removed his private website